src/stp/invariants/DistanceFromBallInvariant.cpp

```diff
@@ -0,0 +1,37 @@
+//
+// Created by timovdk on 4/21/20.
+//
+
+#include "stp/invariants/DistanceFromBallInvariant.h"
+
+namespace rtt::ai::stp::invariant {
+DistanceFromBallInvariant::DistanceFromBallInvariant() noexcept {
+    /**
+     * Creates a piecewise linear function that looks as follows:
+     *
+     * (0,255)  |XX
+     *          | XXXXXX
+     *          |      XXXXXX
+     *          |           XXXXXX
+     *   (0,0)  |----------------XXXXXXX
+     *              (Distance to Ball)
+     */
+    piecewiseLinearFunction = nativeformat::param::createParam(0, 255, 0, "testParam");
```

Comment:
Maybe use FUZZY_TRUE and FUZZY_FALSE for 0 and 255

Reply:
Yes